### PR TITLE
DEVX-1594: Add cp-demo diagram to top of examples

### PR DESCRIPTION
**Pull request by ybyzek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,8 +15,10 @@ This is a curated list of demos that showcase Apache Kafka® event stream proces
 
 ## Where to start
 
-The best demo to start with is [cp-demo](https://github.com/confluentinc/cp-demo) which spins up a Kafka event streaming application using KSQL for stream processing, with many security features enabled, in an end-to-end streaming ETL pipeline with a source connector pulling from live IRC channels and a sink connector connecting to Elasticsearch and Kibana for visualizations.
-`cp-demo` also comes with a playbook and is a great configuration reference for Confluent Platform.
+The best demo to start with is [cp-demo](https://github.com/confluentinc/cp-demo) which spins up a Kafka event streaming application using KSQL for stream processing, with many security features enabled, in an end-to-end streaming ETL pipeline with a source connector pulling from live data and a sink connector connecting to Elasticsearch and Kibana for visualizations.
+`cp-demo` also comes with a tutorial and is a great configuration reference for Confluent Platform.
+
+<br><img src="https://github.com/confluentinc/cp-demo/blob/5.4.1-post/docs/images/cp-demo-overview.jpg" width="420">
```
Comment (rspurgeon) [inline README.md]: Can you center the image?

**Comment by ybyzek** [comment]
@rspurgeon I tried a few permutations to try to center the image, and none of them worked.  Do you want a go at it, or should I merge as-is?

**Comment by rspurgeon** [comment]
> @rspurgeon I tried a few permutations to try to center the image, and none of them worked. Do you want a go at it, or should I merge as-is?

How does the version I just pushed look to you?  I also updated to use the github raw content service which I think is the preferred way to serve/link raw data

**Comment by rspurgeon** [comment]
> > @rspurgeon I tried a few permutations to try to center the image, and none of them worked. Do you want a go at it, or should I merge as-is?
> 
> How does the version I just pushed look to you? I also updated to use the github raw content service which I think is the preferred way to serve/link raw data

However, I didn't update all the other images to use the raw content service, we could maybe do that in another PR

**Comment by ybyzek** [comment]
Wonderful!!  LGTM , thank you :)

**Pull request merged**